lib/haml_lint/tree/tag_node.rb

```diff
@@ -198,5 +206,17 @@ def script
     def text
       (@value[:value] if @value[:parse]) || ''
     end
+
+    private
+
+    def parsed_attributes
+      HamlLint::RubyParser.new.parse(hash_attributes_source)
```

Comment:
There is a `parse_ruby` helper defined on the `Linter` class which you can use instead.